.changeset/eleven-crews-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Changeset is not clear enough, what typings were improved? how? how does it affect end-users?

Reply:
I added a bit more information.